faq.md

```diff
@@ -0,0 +1,82 @@
+
+## How do I use the DITA Open Toolkit?
+
+You can [download][2], [install][3] and [build output][4] from **DITA** content using the command line.
+
+Besides this, there are applications which come with the **DITA Open Toolkit** bundled. For example Oxygen XML Editor usually comes bundled with the latest **DITA Open Toolkit**. Oxygen provides visual means to run the bundled **DITA Open Toolkit** using a concept called [transformation scenarios][5].
+
+## What version of DITA Open Toolkit should I use?
+
+You should try to use the latest **DITA Open Toolkit** release available on the official [download][6] page.
+
+## What outputs can I obtain using the DITA Open Toolkit?
+
+The entire set of default available output formats is available here: [https://www.dita-ot.org/dev/topics/output-formats.html][7]. But the **DITA Open Toolkit** can be enhanced by installing plugins to provide additional output formats.
+
+## What is the general architecture of the DITA Open Toolkit?
+
+The **DITA Open Toolkit** is a quite large mixture of [ANT][8] build scripts, Java libraries and [XSLT scripts][9]. It has a [pipeline-based architecture][10] which uses plugins to publish DITA content to various output formats. Most of the **DITA Open Toolkit** customizations that you want to make in order to add new publishing capabilities or to customize existing publishing choices can be made without modifying its internal core.
+
+## What is a DITA Open Toolkit plugin?
+
+A **DITA Open Toolkit** plugin can either provide a [new publishing format][11], [customize an existing publishing stage][12] or provide a DITA [specialization vocabulary][13]. The plugin can use one of the numerous extension points available in the **DITA Open Toolkit**: [https://www.dita-ot.org/dev/extension-points/plugin-extension-points.html][14].
+
+Once you have [created][15] a plugin you can install it in the **DITA Open Toolkit** either by [manual installation][16] or using the new [automated installation procedure][17].
+
+## How do I customize the HTML-based outputs?
+
+There are a number of parameters which can be set to customize the HTML-based outputs: [http://www.dita-ot.org/dev/parameters/parameters-base-html.html][18]. For example you can specify your own **CSS** stylesheet to be used with the generated HTML output.
+
+You can also create a plugin to customize the HTML outputs by adding a custom XSLT stylesheet: [Creating a simple DITA Open Toolkit plugin to customize published HTML and PDF content][19].
+
+## How do I customize the PDF output?
+
+Oxygen comes bundled with a **DITA Open Toolkit** plugin which uses CSS to style the DITA content and produce PDF: [https://www.oxygenxml.com/doc/ug-editor/topics/pdf-css-customization.html][20]. This plugin is free to use from inside Oxygen but is part of a commercial product if you want to automate the publishing on the server side. This is our recommended plugin for publishing DITA to PDF.
```

Comment:
DITA-OT should not be documenting Oxygen as the first option to customize PDFs. IMO remove this whole paragraph.

Reply:
> our recommended plugin

This absolutely is not our stand.

Reply:
Resolved in 6bf4448.